pkg/templates/sprig_wrapper.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2022 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package templates
+
+import (
+	"github.com/Masterminds/sprig/v3"
+)
+
+var (
+	SprigFuncMap = sprig.FuncMap()
+
+	// ExportedSprigFunctions lists all of the functions from sprig that will be exposed
+	ExportedSprigFunctions = []string{
+		"trim",
+		"upper",
+		"lower",
+		"contains",
+		"hasPrefix",
+		"hasSuffix",
+		"quote",
+		"cat",
+		"replace",
+		"split",
+		"splitn",
+		"list",
+		"join",
+		"until",
+		"untilStep",
+		"default",
+		"empty",
+		"fromJson",
+		"mustFromJson",
+		"ternary",
+		"semver",
+		"semverCompare",
+	}
+)
+
+func GetSprigFunc(funcName string) (result map[int]interface{}) {
```

Comment:
Could you make this a private function?
```suggestion
func getSprigFunc(funcName string) (result map[int]interface{}) {
```